app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/StripePaymentActivity.java

```diff
@@ -52,7 +55,7 @@
     Boolean setSource = false;
     int ticketHistory;
     int eventID;
-    long price = -1; // ticket price IN CENTS
+    double price = -1; // ticket price IN CENTS
```

Comment:
Why is it double then, if its in cents?

Reply:
yes I actually changed it to String now, since we only use that to display it in a textView.

Reply:
But its not a string, coming from the API? How are you going to format a string then? You would need to parse and then format it accordingly. Sense this makes none.